wrangler-core/src/main/java/io/cdap/wrangler/utils/ColumnConverter.java

```diff
@@ -203,8 +205,10 @@ private static Object convertType(String col, String toType, Object object) thro
           return Bytes.toString((byte[]) object);
         } else if (object instanceof Boolean) {
           return Boolean.toString((Boolean) object);
+        } else if (object instanceof LocalDateTime) {
```

Comment:
while we're doing this, should also add one for timestamps (ZonedDateTime)

Reply:
Is a toString() enough on ZonedDateTime since we don't have a specified format?